client/index.html

```diff
@@ -478,7 +478,7 @@ <h1 class="title">Help</h1>
 								</p>
 								<p>
 									A color reference can be found
-									<a href="http://www.mirc.com/colors.html" target="_blank">here</a>.
+									<a href="http://modern.ircdocs.horse/formatting.html#colors" target="_blank" rel="noopener">here</a>.
```

Comment:
@DanielOaks, would you be able to serve these with HTTPS? You have free and easy options now (Let's Encrypt), and it would be a good thing for us to point our users to HTTPS content only. Thanks in advance! 😃